### DISCUSSION TITLE
Input Vietnamese Fcitx in Qutebrowser

### DISCUSSION THREAD

**Discussion by thanhtai9606** [qutebrowser/qutebrowser#8299] [Q&A]
Hi team, 
How can I input Vietnamese by Fcitx in Qutebrower. I cannot do it!
Thanks

**Comment by NguyenThanhDuyOfficial** [comment]
Tui dùng Fcitx bình thường mà, bấm I vào Insert mode rồi Ctrl-Space thui

**Comment by thanhtai9606** [comment]
> Tui dùng Fcitx bình thường mà, bấm I vào Insert mode rồi Ctrl-Space thui

Mình cũng bấm I input mà k nhập được Tiếng Việt trên Qutebrowser cay! Chứ thích xài qute lắm

**Comment by NguyenThanhDuyOfficial** [comment]
Lạ vậy, chứ tui đang dùng Fcitx với qute để rep ông đó.

**Comment by thanhtai9606** [comment]
>  Lạ vậy, chứ tui đang dùng Fcitx với qute để rep ông đó.

Để mình nghiên cứu xem! Trên Mac và wwindow thì chưa thử chứ ẢchLinux mình đang dùng là bị

**Comment by NguyenThanhDuyOfficial** [comment]
Tui dùng ArchLinux + Wayland

**Comment by thanhtai9606** [comment]
> Tui dùng ArchLinux + Wayland

Để mình ngâm cứu thử! Thanks bạn

**Comment by The-Compiler** [comment]
Migrating this to a discussion as this isn't anything actionable for qutebrowser - issues should be used for bugs and feature requests only.

**Comment by Black-233** [comment]
### Solution for Fcitx5 Vietnamese Input in Qutebrowser (Wayland)  
**Environment**: Arch Linux + Wayland + Sway + Fcitx5  

#### Issue:  
Previously failed to input Vietnamese when using:  
```bash
QT_IM_MODULE=fcitx5
```

#### Fix:  
Change the environment variable to:  
```bash
QT_IM_MODULE=wayland
```

#### Source:  
Documented in the [Arch Linux Wiki - Fcitx5](https://wiki.archlinux.org/title/Fcitx5#Software_using_Wayland_input_protocol_cannot_obtain_Wayland_popup_window) under the Wayland input protocol section.
